src/test/integration/intercom/test_task_communication.py

```diff
@@ -63,9 +63,10 @@ def test_single_file_task(self):
         assert task.uid == test_fw.uid, 'uid not transported correctly'
         assert task.scheduled_analysis
 
-    @pytest.mark.skip(reason='TODO')
     def test_re_analyze_task(self):
```

Comment:
I don't understand how this test worked before.
It crashes when trying to create files in `post_processing`.

Reply:
@jstucke Any ideas?

Reply:
Fixed.